ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -787,6 +787,6 @@ Maybe Wait For Dashboard Loading Spinner Page
     END
 
     Run Keyword And Ignore Error    Run Keywords
-    ...    Wait Until Page Contains Element    xpath=//span[@class="pf-c-spinner__tail-ball"]    timeout=${timeout}
+    ...    Wait Until Page Contains Element    xpath=//span[@class="pf-c-spinner__tail-ball"]    timeout=3s
```

Comment:
Is hardcoding a 3s timeout intentional?

Reply:
But why not use the variable in that case?

Reply:
done